modules/svg/image_loader_svg.cpp

```diff
@@ -79,6 +79,12 @@ Error ImageLoaderSVG::create_image_from_utf8_buffer(Ref<Image> p_image, const Pa
 	float fw, fh;
 	picture->size(&fw, &fh);
 
+	// May in some web-based cases return zero size. Render 1x1 px image.
+	if (fw <= 0.0 || fh <= 0.0) {
```

Comment:
Should we make it `< 1.0` instead? Otherwise we have this weird inconsistent state where intermediate values are acceptable, but zero or below makes it go back to 1. If this is pixels, checking against 1 should be correct, IMO.

Reply:
Yes, IMO too - it's safer. Images below point size don't make sense.